common/src/dyn_regs.C

```diff
@@ -34,6 +34,8 @@
 #include "external/rose/rose-compat.h"
 #include "external/rose/powerpcInstructionEnum.h"
 #include "external/rose/armv8InstructionEnum.h"
+#include "external/rose/amdgpuInstructionEnum.h"
+#include <boost/format.hpp>
```

Comment:
@bbiiggppiigg Do we really need to use `<boost/format.hpp>`? @hainest Is there a std c++ equivalent?

Reply:
I did a quick grep and couldn't find any usage of it. I think we can remove it.

> Is there a std c++ equivalent?

Yes, but it requires C++20.